apps/project/graphql/queries.py

```diff
@@ -54,6 +60,21 @@ def _get_vector_tile_server_type(enum: VectorTileServerNameEnumWithoutCustom):
 
 @strawberry.type
 class Query:
+    @strawberry.field(extensions=[IsAuthenticated()])
+    def default_custom_options(self, project_type: ProjectTypeEnum) -> list[CustomOptionType]:
+        custom_option = get_custom_option(project_type=project_type)
+        return [
+            CustomOptionType(
+                title=item["title"],
+                icon=item["icon"],
+                value=item["value"],
+                description=item["description"],
+                icon_color=item["icon_color"],
+            )
+            for item in custom_option
+            if custom_option
```

Comment:
This check does not work if custom_option is None.

Also, let's rename this to custom_options.
And, let's rename the function to get_custom_options